test_runner/fixtures/neon_fixtures.py

```diff
@@ -287,6 +287,7 @@ def conn_options(self, **kwargs):
         if "dsn" in kwargs:
             result.update(parse_dsn(kwargs["dsn"]))
         result.update(kwargs)
+        result = {k: v for k, v in result.items() if v is not None}
```

Comment:
Curious what can yield None value. Is it password?

Reply:
Yep, I use it to drop the password from the connstr (when it's set in default_options).
Worth to add a comment for it.